apps/api-extractor/src/extractor/Extractor.ts

```diff
@@ -63,6 +63,14 @@ export interface IExtractorOptions {
    * The default value is false.
    */
   localBuild?: boolean;
+
+  /**
+   * If specified, use typings specified in the project's compilerOptions -> lib option
```

Comment:
>If [](start = 5, length = 2)

Clarify that this is a path to the TypeScript compiler's package folder.  I didn't get that from the text.